RevenueCatUI/Helpers/Package+VariableDataProvider.swift

```diff
@@ -15,6 +15,34 @@ import RevenueCat
 @available(iOS 15.0, macOS 12.0, tvOS 15.0, *)
 extension Package: VariableDataProvider {
 
+    var shouldRoundPrices: Bool {
+        guard let countryCode = Purchases.shared.storeFrontCountryCode else {
+            return false
+        }
+        let trimCentsStoreFrontCountryCodes: Set<String> = [
+                "TWN", // Taiwan
+                "KAZ", // Kazakhstan
+                "MEX", // Mexico
+                "PHL", // Philippines
+                "THA" // Thailand
+            ]
+        return trimCentsStoreFrontCountryCodes.contains(countryCode)
+    }
+
+    var localeByStorefrontCountryCode: [String: Locale] {
+        guard let countryCode = Purchases.shared.storeFrontCountryCode else {
+            return [:]
+        }
+
+        return [
+            "TWN": Locale(identifier: "zh_Hant_TW"), // Taiwan
+            "KAZ": Locale(identifier: "kk_Cyrl_KZ"), // Kazakhstan
+            "MEX": Locale(identifier: "es_MX"), // Mexico
+            "PHL": Locale(identifier: "fil_PH"), // Philippines
+            "THA": Locale(identifier: "th_TH"), // Thailand
+        ]
+    }
```

Comment:
Will that API need to give both the strong front and locale identifier to the SDK for this? 👀 

And note that the countries/storefronts that will need to be formatted will be coming in from `PaywallData` so this might need to be passed into a function somewhere

Reply:
> Will that API need to give both the strong front and locale identifier to the SDK for this? 👀

Ya, I'll see if this can be avoided. If not, a mapping will need to exist _somewhere_...

> And note that the countries/storefronts that will need to be formatted will be coming in from PaywallData so this might need to be passed into a function somewhere

Ya I knew this was temporary, just as far as I'd gotten at this point. I'll progress this to look in PaywallData.

Reply:
Looks unlikely we'll be able to fish out a locale in this situation as per [code/comments here.](https://github.com/RevenueCat/purchases-ios/blob/781151df49f570b070c02961ccdfc1147707be5d/Sources/Purchasing/StoreKitAbstractions/SK2StoreProduct.swift#L91-L122).

Reply:
Oh nooooo... I guess we'll have to include that in the API response 😬